src/types/qwinputdevice.cpp

```diff
@@ -6,7 +6,7 @@
 #include "qwkeyboard.h"
 #include "qwpointer.h"
 #include "qwtablet.h"
-#include "qwtabletpad.h""
```

Comment:
Can we add a github bot to check code format，for example https://github.com/marketplace/codefactor

Reply:
Try it.